server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -64,8 +73,36 @@ pub struct RemoteDataSynchroniser {
 
 impl RemoteDataSynchroniser {
     /// Request initialisation
+    /// TODO improve logic for initialisation check, requires central changes as per this issue:
+    /// https://github.com/openmsupply/open-msupply-internal/issues/30#issuecomment-1567639141
```

Comment:
There are some details of how this can be done better, I think what's here now would work ok, the only issue is that it's possible (very small chance) for initialisation to be skipped